### ISSUE TITLE
Nothing append after loading the batterystats.txt file in the docker app (or after execute python script)

### ISSUE THREAD

**Issue by JejeF** [google/battery-historian#117]
Hello, 

- I have a trouble to use battery-historian app with docker  when Ioad "batterystats.txt" following the procedure describe in https://developer.android.com/studio/profile/battery-historian.html. I just load the file and nothing append.

- I also try also the python script :  python historian.py -p batterystats.txt > batterystats.html , there is just this line in the output html file : "Battery history not present in bugreport" .

Could somebody give me a good "batterystats.txt' file to test the battery-historian app and tell me where is the problem.
[batterystats.txt](https://github.com/google/battery-historian/files/970895/batterystats.txt)

Thanks for your help

**Comment by NNickey** [comment]
Just to alert you early, bug reports or battery stats can contain personal information such as email, the apps you use, gps location, wifi access points you use, and even certain notifications from apps you use.

So far, it seems you havn't privacy cencored your uploaded txt file.
Just a heads up.

**Comment by WeiAlawn** [comment]
Hi there, I have faced the same issue that cannot get diagrams, only displayed text information which is same as batterystats.txt, even run python script. Could U help me how to solve this problem. 
Thanks

**Comment by JejeF** [comment]
@WeiAlawn do you have tried  to use use the docker image and load a bugreport.txt file instead ? 
Information are in the readme  https://github.com/google/battery-historian

**Comment by WeiAlawn** [comment]
Yes, I've tried run on Docker, when it's setup and type the right URL of historian, after that upload the bugreport.txt then submit. Didn't show up diagrams as well. Tried this several times, also someone told me try VPN(ps:Google services don't work in Chinese networks) but it won't work and got 502 proxy error.
